.github/workflows/iroha2-release.yml

```diff
@@ -52,10 +52,28 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
-  load-rs:
+  configs:
     runs-on: ubuntu-latest
     container:
       image: hyperledger/iroha2-ci:nightly-2022-12-22
+    steps:
+      - uses: actions/checkout@v3
+        ref: iroha2-dev
+      - name: Setup git config
+        run: |
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+      - name: Update configs
+        run: |
+          ./scripts/update_configs.sh lts
+          ./scripts/update_configs.sh stable
+      - name: Commit config changes
+        run: |
+          git commit -am "[documentation]: Update lts/stable configs following a release" --signoff
+          git push origin iroha2-dev
+
+  load-rs:
```

Comment:
I'd skip this (or allow failures) until @BAStos525 fixes the workflow on the side of the load program. False-positives are a problem.